src/charm.py

```diff
@@ -29,7 +29,8 @@
 ]
 METRICS_PATH = "/metrics"
 METRICS_PORT = "8080"
-WEBHOOK_PORT = "9443"
+WEBHOOK_PORT = "443"
+WEBHOOK_TARGET_PORT = "9443"
```

Comment:
Doesn't this mess with L54? We were already using webhook_port but for 9443

Reply:
Oh, let me correct it, good catch!